### DISCUSSION TITLE
Help! How to undo uploaded change

### DISCUSSION THREAD

**Discussion by Seanymph-RS** [htool/vhfinfo#13] [Q&A]
This morning I have edited a VTS-area (VTS Den Helder) and that change is now implemented in the map. But somehow, don´t ask me how, I seem to have deleted another (very important) adjoining area (Centrale Meldpost Waddensea). How to undo this giant f#@%k up?

**Comment by htool** [comment]
I'll check it out today. It's all in git, so nothing lost.
